newsletter-june-2023.rst

```diff
@@ -0,0 +1,71 @@
+.. post:: June 1, 2023
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+   :category: Newsletter
+
+Read the Docs newsletter - June 2023
+====================================
+
+News and updates
+----------------
+
+- ⚠️ A ``.readthedocs.yaml`` configuration file will be required for your future builds.
+  Read more about this change in :doc:`/migrate-configuration-v2`.
+- ✅️ Visiting ``/en/`` on a documentation project now redirects to ``/en/latest/``.
+- 🍿️ All the talks from `Write the Docs Portland 2023 <https://www.writethedocs.org/conf/portland/2023/>`__ are ready to watch:
+  `Open the new playlist on Youtube <https://www.youtube.com/watch?v=EZJ0mk9Jj3s&list=PLZAeFn6dfHpneQPsDWa4OmEpgW4pNiaZ2>`__.
+- 🐛️ Builds with multiline commands suffered from `a few bugs <https://github.com/readthedocs/readthedocs.org/issues/10103>`__ that are hopefully all fixed now.
+  Thanks to everyone who helped out!
+- 🐛️ `sphinx-rtd-theme 1.2.1 <https://sphinx-rtd-theme.readthedocs.io/en/stable/changelog.html>`__ has been released with an important bug fix that caused jQuery to not load in certain projects.
+- 🔒️ We updated custom domain certificates to 1.2 as the *minimum* TLS version.
+- 🔒️ A high-severity vulnerability has been fixed:
+  `Write access to projects via API V2 (/api/v2/project/* endpoints) for any logged-in user <https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-rqfv-8rrx-prmh>`__.
+
+Possible issues
+---------------
+
+- ⚠️ Please make sure to read the blog post: :doc:`/migrate-configuration-v2`.
+
+  If you didn't have a ``.readthedocs.yaml`` configuration file,
+  and you are introducing one for the first time,
+  chances are that a lot of your documentation stack is outdated (Python, Sphinx, Ubuntu).
+  So your builds might break.
```

Comment:
```suggestion
  and you are introducing one for the first time,
  we're excited for you to be able to use all our new features like specifying build tool versions (Node, Rust, etc.)!
```

Reply:
Ah yes, that's more positive. Always hard to choose between the negative reasoning for taking action and the positive reasoning for making changes.